src/index.js

```diff
@@ -77,6 +77,23 @@ const main = async () => {
     }
     res.status(201).send({ postId: id })
   })
+  //vote posts
+  app.put('/votes/:id', async (req,res)=>{
```

Comment:
You are missing a space between `req` and `res`. You should run `yarn run format` and `yarn run lint` before committing.